syncapi/sync/notifier_test.go

```diff
@@ -124,22 +126,33 @@ func TestNewEventAndJoinedToRoom(t *testing.T) {
 	var wg sync.WaitGroup
 	wg.Add(1)
 	go func() {
-		pos, err := waitForEvents(n, newTestSyncRequest(bob, syncPositionBefore))
+		pos, err := waitForEvents(n, newTestSyncRequest(bob, bobDev, syncPositionBefore))
 		if err != nil {
 			t.Errorf("TestNewEventAndJoinedToRoom error: %w", err)
 		}
 		mustEqualPositions(t, pos, syncPositionAfter)
 		wg.Done()
 	}()
 
-	stream := lockedFetchUserStream(n, bob)
+	stream := lockedFetchUserStream(n, bob, bobDev)
 	waitForBlocking(stream, 1)
 
 	n.OnNewEvent(&randomMessageEvent, "", nil, syncPositionAfter)
 
 	wg.Wait()
 }
 
+func TestCorrectStream(t *testing.T) {
```

Comment:
Please add a test for 1 user with 2 devices to make sure waking up 1 device doesn't wake up the other.

Reply:
Done.